db/migrate/20220907145552_clean_trends_and_foreman_docker_data.rb

```diff
@@ -0,0 +1,14 @@
+require_dependency File.expand_path('cleanup_helper', './lib/tasks/')
+
+include CleanupHelper
+
+class CleanTrendsAndForemanDockerData < ActiveRecord::Migration[6.1]
+  def up
+    clean_foreman_docker
```

Comment:
I think in packaging we should make foreman.rpm obsolete the foreman_docker package so we're sure it's not installed anywhere.

Reply:
I agree to that! https://projects.theforeman.org/issues/35538